src/fields/field_fe.cc

```diff
@@ -97,14 +115,14 @@ typename Value::return_type const & FieldFE<spacedim, Value>::value(const Point
 		if (dh_->fe<1>()->is_scalar()) {
```

Comment:
Make a protected method template (with element dimension as parameter) for evaluation in order to have common implementation for all element dimensions. Check that it works also for dim==0, which should resolve problems with usage on boundary.